app/main.py

```diff
@@ -1,6 +1,20 @@
 from typing import Callable
+import functools
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = dict()
+
+    @functools.wraps(func)
+    def cache_wrapper(*args, **kwargs) -> Callable:
```

Comment:
The wrapped function does not always return `Callable`, replace the return type annotation with `Any` to have an actual representation.